docs/cli.rst

```diff
@@ -0,0 +1,331 @@
+The ``cci`` Command Line
```

Comment:
This document as a whole is great but one thing feels missing: any mention of interacting with an org. Not a blocker for release, but I'd like to add a short subsection at least pointing users to other sections where that is covered.